apps/src/code-studio/components/AssetRow.jsx

```diff
@@ -185,40 +117,34 @@ export default class AssetRow extends React.Component {
       flex = size + ' kb';
     }
 
+    let usage = $('#visualization').find(
+      `[src*=\'${encodeURIComponent(this.props.name).replace("'", "\\'")}']`
```

Comment:
@jmkulwik this fix looks valid, but it might be even simpler to just use double quotes: 
`[src*="${encodeURIComponent(this.props.name)}"]`. docs are not the clearest: https://api.jquery.com/attribute-contains-selector/